apimartifacts/diagnostics/applicationinsights/diagnosticInformation.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "alwaysLog": "allErrors",
+    "backend": {
+      "request": {
+        "dataMasking": {
+          "queryParams": [
+            {
+              "mode": "Hide",
+              "value": "*"
+            }
+          ]
+        }
+      }
+    },
+    "frontend": {
+      "request": {
+        "dataMasking": {
+          "queryParams": [
+            {
+              "mode": "Hide",
+              "value": "*"
+            }
+          ]
+        }
+      }
+    },
+    "httpCorrelationProtocol": "Legacy",
+    "logClientIp": true,
+    "loggerId": "/subscriptions/{SUBSCRIPTION_ID}/resourceGroups/rg-pktmtbrc-dev/providers/Microsoft.ApiManagement/service/apim-pktmtbrc-dev/loggers/applicationinsights",
```

Comment:
Should this go in some config file too? Happy for this PR to go in and we solve with another task (feel free to resolve)